IPs/RSKIP143.md

```diff
@@ -0,0 +1,42 @@
+# Segwit Compatibility
+
+|RSKIP          |143          |
+| :------------ |:-------------|
+|**Title**      |Segwit Compatibility|
+|**Created**    |22-OCT-19 |
+|**Author**     |PGP |
+|**Purpose**    |Sca,USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft|
+
+## Abstract
+
+This document describes the procedure allows the 2wp proccess to accept segwit transaction sent using a P2SH-PWPKH wallet. 
```

Comment:
This document describes the procedure ~~allows~~ **to allow** the 2wp pro~~c~~cess to accept segwit transaction**s** sent using a P2SH-PWPKH wallet.